tests/countries/test_india.py

```
@@ -982,3 +981,10 @@ def test_deprecated(self):
         self.assertEqual(
             India(subdiv="OR", years=2023).keys(), India(subdiv="OD", years=2023).keys()
         )
+    def test_maharashtra_2026(self):
+        in_holidays=holidays.India(subdiv="MH", years=2026)
+        self.assertIn("2026-02-19", in_holidays)
```

Ruff (0.15.6)

[warning] Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)